From: Stuart Prescott <stuart@debian.org>
Date: Fri, 3 May 2024 16:50:13 +1000
Subject: Add failing tests to blacklist

Tests are still run but will fail. They are run multiple times to see if
they still fail - it would be better to just skip them.

It is not yet clear whether these failures are important.
---
 build_history/blacklist.txt | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/build_history/blacklist.txt b/build_history/blacklist.txt
index 5f0183f..f805980 100644
--- a/build_history/blacklist.txt
+++ b/build_history/blacklist.txt
@@ -91,3 +91,19 @@
 [QtQml::qqmlincubator_incubateWhile]
     pypy        # gives timeout on macOS if I run video converter while testing
 # 2022-03-04 PyPy: Currently 28 of 657 tests fail
+
+# Debian packaging failing tests
+[sample::array_numpy]
+    py3
+[sample::nontypetemplate]
+    py3
+[pyside6-android-deploy::test_pyside6_android_deploy]
+    py3
+[QtCore::qinstallmsghandler_test]
+    py3
+[QtCore::loggingcategorymacros_test]
+    py3
+[QtGui::qpainter_test]
+    py3
+[QtCharts::qcharts_numpy_test]
+    py3
